Client2.py

```diff
@@ -1,45 +1,58 @@
-"""Client Class File"""
+"""This file consists of the client class, its respective methods and their implementations.
+   The client is designed to send, receive data and echo the data simultaneously."""
 
 import socket
+import logging
 
+zero = 0        # Global Variable for zero
+limit = 16      # Data limit
+
+logging.basicConfig(filename='client.log', level=logging.INFO)
 
 class Client:
-    """comment"""
+    """Client class defines the instances and methods required for:
+    1) Creating socket
+    2) Sending Data
+    3) Receiving Data
+    4) Closing the transmission. """
 
     def __init__(self, ip, port):
-        """comment"""
+        """init method defines the necessary instances:
```

Comment:
Please read this section: https://google.github.io/styleguide/pyguide.html?showone=Comments#Comments